BuildResidentialScheduleFile/tests/test_build_residential_schedule_file.rb

```diff
@@ -279,6 +294,26 @@ def test_zero_occupants
     assert_empty(hpxml.buildings[0].header.schedules_filepaths)
   end
 
+  def test_different_time_zone_latitude_longitude
```

Comment:
How about a test like `test_stochastic` but using the new `base-location-detailed.xml` sample file?